testing/environments/snapshot-oss.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.0-SNAPSHOT
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.8.1-SNAPSHOT
```

Comment:
We just branched off 7.8 for the 7.8.0 release. The version bump to 7.8.1 will happen only after 7.8.0 is released. For now we need to keep this at `7.8.0-SNAPSHOT`. This will get us the 'nightlies' of the ES 7.8 release branch even before BCs become available.

Reply:
done